src/Castle.Windsor/Core/Internal/ReflectionUtil.cs

```diff
@@ -189,9 +189,10 @@ private static Assembly LoadAssembly(AssemblyName assemblyName)
 			return Assembly.Load(assemblyName);
 		}
 #endif
-		public static TAttribute[] GetAttributes<TAttribute>(this MemberInfo item) where TAttribute : Attribute
+
+		public static TAttribute[] GetAttributes<TAttribute>(this MemberInfo item, bool inherit) where TAttribute : Attribute
 		{
-			return (TAttribute[])Attribute.GetCustomAttributes(item, typeof(TAttribute), true);
+			return (TAttribute[])Attribute.GetCustomAttributes(item, typeof(TAttribute), inherit);
```

Comment:
I changed this to avoid ambiguity because [Castle.Core](https://github.com/castleproject/Core/blob/master/src/Castle.Core/Core/Internal/AttributesUtil.cs) also has a GetAttributes() extension method, and it passes `false` for `inherit`